examples/hitl/rearrange/config/hitl_rearrange.yaml

```diff
@@ -2,22 +2,40 @@
 
 defaults:
   - social_rearrange: pop_play
+  - hitl_defaults
   - _self_
 
 habitat:
+  # todo: document these choices
   environment:
+    max_episode_steps: 0
     iterator_options:
       cycle: False
+  task:
+    measurements:
+      pddl_success:
+        must_call_stop: False
 
 habitat_baselines:
-  evaluate: True
-  num_environments: 1
+  # todo: document these choices
   eval:
     should_load_ckpt: False
   rl:
     agent:
       num_pool_agents_per_type: [1,1]
 
+habitat_hitl:
+  window:
+    title: "Rearrange"
+    width: 1300
+    height: 1000
+  gui_controlled_agent:
+    agent_index: 1
+  data_collection:
```

Comment:
I'm turning on data-collection for the rearrange example app because it's an important feature of this app to demonstrate.